docs/user-guide/src/capm3/pivoting.md

```diff
@@ -0,0 +1,167 @@
+# CAPM3 Pivoting
+
+## What is pivoting
+
+Cluster-api-provider-metal3 (CAPM3) implements support for CAPI's 'move/pivoting' feature.
+
+CAPI Pivoting feature is a process of moving the provider components and declared Cluster API resources from a source
+management cluster to a target management cluster by using the `clusterctl` functionality called "move".
+More information about the general CAPI "move" functionality can be find [here](https://cluster-api.sigs.k8s.io/clusterctl/commands/move.html).
+
+In Metal3, pivoting is performed by using the CAPI `clusterctl` tool provided by Cluster-API. `clusterctl` recognizes pivoting as move.
+During the pivot process `clusterctl` pauses any reconciliation of Cluster-API objects and this gets propagated to (CAPM3) objects as well.
+Once all the objects are paused, the objects are created on the other side on the target cluster and deleted from the
+source cluster.
+
+## Prerequisits
+
+* **It is mandatory to use `clusterctl` for both the bootstrap and target cluster.**
+
+  If the provider components are not installed using `clusterctl`, it will not
+  be able to identify the objects to move.  Initializing the cluster using
+  `clusterctl` essentially adds the following labels in the CRDs of each related object.
+
+  ```yaml
+  labels:
+  - clusterctl.cluster.x-k8s.io: ""
+  - cluster.x-k8s.io/provider: "<provider-name>"
+  ```
+
+  So if the clusters are not initialized using `clusterctl`, all the CRDS of the
+  objects to be moved to target cluster needs to have these labels both in
+  bootstrap cluster and target cluster before performing the move.
+
+  **Note**: This is not recommended, since
+  the way `clusterctl` identifies objects to manage might change in the future, so
+  it's always safe to install CRDs and controllers through the `clusterctl init` sub-command.
+
+* **BMH objects have correct status annotation.**
+
+  Since BMH _status_ holds important information regarding the BMH itself, BMH with
+  status has to be moved and BMH has to be reconstructed with correct status in
+  target  cluster before it is being reconciled. This is now done through BMH
+  status annotation in BMO.
```

Comment:
you could provide a reference to BMO doc here

Reply:
I would rather have somebody add it later when the bmo documentation is part of the user-guide.

Reply:
Is there an open issue to add it to the user guide? If not, why not to reference it

Reply:
Yes there is https://github.com/metal3-io/metal3-docs/issues/221, but based on the PR https://github.com/metal3-io/metal3-docs/pull/237 I can preemptively place the link, I hope the file names won't change.